src/commands/pwd.rs

```diff
@@ -1,9 +1,9 @@
 use crate::commands::WholeStreamCommand;
+// use crate::data::{Dictionary, Value};
```

Comment:
Could you remove this comment?

Reply:
Yeah, I thought I got them all, but missed a couple, I will remove them.

Reply:
Thanks!